validator_test.go

```diff
@@ -136,6 +137,30 @@ func TestNewValidator_ValidateDocument(t *testing.T) {
 	assert.Len(t, errs, 0)
 }
 
+type alwaysMatchesRegex jsonschema.RegexpEngine
+
+func (dr *alwaysMatchesRegex) MatchString(s string) bool {
+	return true
+}
+
+func (dr *alwaysMatchesRegex) String() string {
+	return ""
+}
+
+func fakeRegexEngine(s string) (jsonschema.Regexp, error) {
+	return (*alwaysMatchesRegex)(nil), nil
+}
+
+func TestNewValidator_WithRegex(t *testing.T) {
+
+	doc, _ := libopenapi.NewDocument(petstoreBytes)
```

Comment:
Here also check the second value returned

Reply:
Yep .. that's a cut-and-paste from prior code ... :-)